diff.go

```diff
@@ -32,7 +33,7 @@ func getWalkerFn(root string) walkerFn {
 			}
 
 			p := &currentPath{
-				path: path,
+				path: filepath.FromSlash(path),
```

Comment:
I don't think this is correct if it is compared to the `stat` that is coming from protobuf.

All the paths on the proto wire and public types in this library should be unix style. System variant is only used when opening a file from local system (I think this is all internal behavior). When there is communication to the remote side via send/receive/diskwriter etc., what OS remote side is should be irrelevant.

Reply:
Reverted the change. It was handled elsewhere.